oshi-core/src/main/java/oshi/hardware/platform/unix/solaris/SolarisDisks.java

```diff
@@ -52,6 +52,7 @@ public static boolean updateDiskStats(HWDiskStore diskStore) {
             diskStore.setWrites(data.writes);
             diskStore.setReadBytes(data.nread);
             diskStore.setWriteBytes(data.nwritten);
+            diskStore.setQueueLength(data.rlentime / 1000000L);
```

Comment:
Reading the docs this is cumulative run time, which is not the same as queue length.   Should you not use `wcnt` (count of elements in wait state) or `rcnt` (count of elements in run state)?  I'm not sure this stat is consistent with the values you're getting from other OS's